mod/lostpass.php

```diff
@@ -53,7 +53,7 @@ function lostpass_post(App $a) {
 		provided and ignore and/or delete this email.
 
 		Your password will not be changed unless we can verify that you
-		issued this request.'));
+		issued this request.', $user['username'], $sitename));
```

Comment:
I think you should add the hint that the link is only valid for a limited time.

Reply:
Good point.